bmwqemu.pm

```diff
@@ -42,6 +42,9 @@ our @EXPORT_OK = qw(diag);
 use backend::driver;
 require IPC::System::Simple;
 
+my $log_module = "Mojo::Log";
+eval 'use Mojo::Log::Colored; $log_module = "Mojo::Log::Colored"';
```

Comment:
Maybe some of the Perl people know something better?

Reply:
Why the `eval`? Can't we just always use it?

Reply:
Not when we see it as recommendation instead of requirement. The package is so far not even in openSUSE:Factory pending inclusion in [d:l:p](https://build.opensuse.org/request/show/805622) first. I am merely applying the same approach you used for Test::More::Color in openQA. Would you prefer an alternative that is not using Mojo::Log::Colored at all and just relying on Term::ANSIColor?

Reply:
Change 2. seems trivial enough to avoid introducing a dependency for that.

Reply:
ok, done